### ISSUE TITLE
Model Optimization for Calf Muscle Segmentation

### ISSUE THREAD

**Issue by MaryamKargaran** [MuscleMap/MuscleMap#82]
Dear MuscleMap team,

We are currently using the MuscleMap whole-body model for calf muscle segmentation and would like to optimize it further for our specific data. Some of the issues we are facing include label ambiguity in boundary regions between adjacent muscles, as well as the need for more detailed labels, for example, distinguishing between the medial and lateral heads of the gastrocnemius.

Could you advise on the best path forward? Specifically, is there a straightforward way to fine-tune or extend the existing model weights for our use case, or is this something you could support?

Best, 
Maryam

**Comment by Eddowesselink** [comment]
Hi @MaryamKargaran , thanks for reaching out and using MuscleMap. We're currently updating our model for calf/lowerleg segmentations, including the medial and lateral heads of the gastrocnemius, as well as including the tibilias anterior, posterior, flexor digiturm longus, flexor hallucis longus and the extensor hallucis / digitorum longus (combined), popliteus and plantaris.  We also expect higher accuracy due to improvements in the (accuracy) of the training data as well as training diversity. We expect to publish this model at the end of this month. If this not interferes with your timeline, you may could use a preliminary model we will likely open somewhere in the next two weeks in a seperate branch. If you are able to share some data, I can include it in the next sprint so that the intermediate model will better generalize to your dataset.

Cheers,
Eddo

**Comment by MaryamKargaran** [comment]
Hi @Eddowesselink!
Thanks for the reply! unfortunately, there are some issues at our site which makes data sharing a bit complicated! 
Nevertheless, I wanted to check in about the new model with extended labels. I've looked through the repo but couldn't find the separate branch. Could you point me to it?

**Comment by Eddowesselink** [comment]
Hi @MaryamKargaran,

No problem. We're in the last stage of the training procedure. I expect to publish our v1.4 branch soon and will give you a heads-up when it's ready.

Cheers,
Eddo
